@giorgiosironi / @cory 

Have we encountered this before? Looks like it errors with 

```
 ##[error]The input 'submodules' is not supported in actions/checkout@v2
```

This appears to be because we are attempting to use ssh for submodule on a private repo... there is a soln here: https://github.com/actions/checkout/issues/14#issuecomment-553383594

--- a/.github/workflows/publish.yml
+++ b/.github/workflows/publish.yml
@@ -15,8 +15,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-        with:
-          submodules: true
       - name: Build Docker image
         run: make build
       - name: Push image

--- a/.github/workflows/publish.yml
+++ b/.github/workflows/publish.yml
@@ -5,7 +5,7 @@ on:
     types: [opened, synchronize, reopened]
   push:
     branches:
-      - master events but only for the master branch
+      - master
 
 env:
   IMAGE_TAG: ${{ github.sha }}

--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,8 @@
 IMAGE_NAME=reviewer-reporter
 
 build:
+	git submodule sync
+	git submodule update --init --recursive
 	docker build -t ${IMAGE_NAME} .
 
 shell:

